.babelrc

```diff
@@ -1,3 +1,7 @@
 {
-  "presets": ["es2015", "react"]
+  "sourceType": "unambiguous",
+  "presets": [
```

Comment:
I was able to run [`babel-upgrade`](https://github.com/babel/babel-upgrade) successfully, and it took care of this file (mostly) and the package.json file